### PR TITLE
Attempt to fix #27 (path issues on Windows, maybe just too following the exception)

### LINKED ISSUE #27: Windows: cmd.exe + quotes = Hell
In Windows 8.1, with Node.js 5.4.0, running

`node tools\fable2babel.js --projFile "C:\Temp\MyProj\Core\MyProj.Core.fsproj"
`

gives me this

```
cmd /C C:\Temp\Fable\tools/../build/main/Fable.exe {"lib":".","outDir":".","symbols":[],"watch":false,"projFile":"MyProj.Core.fsproj"}
events.js:141
      throw er; // Unhandled 'error' event
      ^

Error: spawn cmd ENOENT
    at exports._errnoException (util.js:856:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:178:32)
    at onErrorNT (internal/child_process.js:344:16)
    at nextTickCallbackWith2Args (node.js:478:9)
    at process._tickCallback (node.js:392:17)
    at Function.Module.runMain (module.js:432:11)
    at startup (node.js:141:18)
    at node.js:1003:3

```

which is not a really useful error message :) -- But then, I tried

`cmd /C C:\Temp\Fable\tools/../build/main/Fable.exe {"lib":".","outDir":".","symbols":[],"watch":false,"projFile":"MyProj.Core.fsproj"}
`

which outputs a somewhat better description of the real problem

```
Unhandled Exception: Newtonsoft.Json.JsonReaderException: Input string '.' is not a valid number. Path 'lib', line 1, position 6.
   at Newtonsoft.Json.JsonTextReader.ParseNumber(ReadType readType)
   at Newtonsoft.Json.JsonTextReader.ReadStringValue(ReadType readType)
   at Newtonsoft.Json.JsonTextReader.ReadAsString()
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.ReadForType(JsonReader reader, JsonContract contract, Boolean hasConverter)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.ResolvePropertyAndCreatorValues(JsonObjectContract contract, JsonProperty containerProperty, JsonReader reader, Type objectType)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateObjectUsingCreatorWithParameters(JsonReader reader, JsonObjectContract contract, JsonProperty containerProperty, ObjectConstructor`1 creator, String id)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateNewObject(JsonReader reader, JsonObjectContract objectContract, JsonProperty containerMember, JsonProperty containerProperty, String id, Boolean& createdFromNonDefaultCreator)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateObject(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateValueInternal(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.Deserialize(JsonReader reader, Type objectType, Boolean checkAdditionalContent)
   at Newtonsoft.Json.JsonSerializer.DeserializeInternal(JsonReader reader, Type objectType)
   at Newtonsoft.Json.JsonConvert.DeserializeObject(String value, Type type, JsonSerializerSettings settings)
   at Newtonsoft.Json.JsonConvert.DeserializeObject[T](String value, JsonSerializerSettings settings)
   at Newtonsoft.Json.JsonConvert.DeserializeObject[T](String value)
   at Fable.Main.main(String[] argv) in C:\Temp\Fable\src\Main.fs:line 73

```

I can't see how that json argument could work at all in Windows. Those double quotes in json would be parsed by the command line processor (cmd.exe). And trying to escape them would be... troublesome (to say the least -- See http://stackoverflow.com/a/15262019 and http://blogs.msdn.com/b/twistylittlepassagesallalike/archive/2011/04/23/everyone-quotes-arguments-the-wrong-way.aspx )

Or I am missing something?

### PR DESCRIPTION
**Pull request by davidpodhola**
Try to make sure `com.Options.projFile` can be passed to `FSharp.Compiler.Service` plus also

- print the current working dir of `Fable.exe`
- use `Global` as module name for anything outside of a module in TypeScript instead of `Unknown`

### CODE REVIEW

**Comment by davidpodhola** [comment]
sorry for the commit mess, will stick to branch-per-feature now to make it easier

File: src/Main.fs [hunk 1]
```diff
@@ -41,8 +41,18 @@ let parseFSharpProject (com: ICompiler) (checker: FSharpChecker) (projCode: stri
         | ".fsx" ->
             let otherFlags =
                 com.Options.symbols |> Array.map (sprintf "--define:%s")
+            let comOptionsprojFile =
+              try
+                let test = Path.GetDirectoryName( com.Options.projFile )
```
Comment (davidpodhola) [inline src/Main.fs]: See [the exception detail](https://gist.github.com/davidpodhola/44c7a1bfbbc92a0da708) why `Path.GetDirectoryName` is used

**Comment by alfonsogarciacaro** [comment]
The solution looks a bit hacky, I'd prefer if we don't need to deal with the issue in Fable's code but in the fable2babel.js script instead.

**Comment by davidpodhola** [comment]
OK, you are right, will have a look at that.

**Pull request closed without merging**